docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -49,6 +49,7 @@ Key Features and Updates
   * FEAT-#4619: Integrate mypy static type checking (#4620)
 * New Features
   * FEAT-4463: Add experimental fuzzydata integration for testing against a randomized dataframe workflow (#4556)
+  * FEAT-#4147: Support multiple pandas versions (#4301)
```

Comment:
Is it possible to write here more informatively which versions of python/pandas are now available for users?